libs/@hashintel/design-system/src/entities-graph-chart.tsx

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable no-restricted-imports */
+import { BaseUrl } from "@blockprotocol/type-system";
+import { Chart, EChart, ECOption } from "@hashintel/design-system";
+import { generateEntityLabel } from "@local/hash-isomorphic-utils/generate-entity-label";
+import {
+  Entity,
+  EntityId,
+  EntityRootType,
+  Subgraph,
+} from "@local/hash-subgraph";
+import { getEntities, getEntityTypeById } from "@local/hash-subgraph/stdlib";
+import {
+  extractBaseUrl,
+  LinkEntity,
+} from "@local/hash-subgraph/type-system-patch";
```

Comment:
You should be able to use equivalents in `@blockprotocol/graph` (to replace `@local/hash-subgraph`) and `@blockprotocol/type-system/slim` (to replace `@local/hash-subgraph/type-system-patch`) to avoid importing from local packages in a published one

Reply:
I've moved the component into the `@hashintel/blocks-design-system` package, and removed the local dependencies which I could. Some remain in the `generateEntityLabel` implementation because of the `simplifyProperties` usage.